lib/test/poseidon/producer.rb

```diff
@@ -0,0 +1,11 @@
+module Test
+  module Poseidon
+    class Producer
+      def initialize(*args)
+      end
+
+      def send_messages(arr)
```

Comment:
Unused method argument - `arr`. If it's necessary, use `_` or `_arr` as an argument name to indicate that it won't be used. You can also write as `send_messages(*)` if you want the method to accept any arguments but don't care about them.